src/ttsnake.c

```diff
@@ -326,8 +326,12 @@ void init_game () {
 /* Generates energy_block[0] coordinates randomly. */
 
 void generate_energy_block () {
-  energy_block[0].x = (rand() % (NCOLS - 2)) + 1;
-  energy_block[0].y = (rand() % (NROWS - 2)) + 1;
+  energy_block[0].x = (rand() % (NCOLS - 2)) + HORIZONTAL_MOVE;
+  energy_block[0].y = (rand() % (NROWS - 2)) + VERTICAL_MOVE;
+
+  /* Verifies if the energy is in a pair position, 'cause the snake moves 2 pos horizontally */
+  if ((energy_block[0].x)%2 != 0)
```

Comment:
Please utilize the coding style that was agreed upon (see discussion at #13). 
```suggestion
  if ((energy_block[0].x)%2 != 0) {
    energy_block[0].x -= 1;
  }
}
```

Reply:
commited